src/github/installation.ts

```diff
@@ -69,7 +75,7 @@ export const installationWebhookHandler = async (
 		);
 
 	} catch (err) {
-		logger.warn({ err }, "Failed to submit security workspace to Jira via backfill");
+		logger.warn({ err }, "Failed to submit security workspace to Jira or trigger backfill via backfill");
```

Comment:
nit: I'm confused with the trigger backfill via backfill. Shouldn't it be clearer if it's "Failed to submit security workspace to Jira via backfill or trigger backfill" ?

Reply:
Will update the message in follow-up PRs